enlace/reports/num_programs.php

```diff
@@ -96,33 +114,58 @@ function(idx, display) {
       </div>
     <div>
 
-    <input type="checkbox" id="select_all_program_checkboxes"> <b>Select All</b><br>
+    <input type="checkbox" id="select_all_recent_program_checkboxes"> <b>Select All Recent</b><br>
 <?php
     $checked_programs = [];
-    $program_string = " WHERE Programs.Program_ID = " . $access_array[0];
+    $program_string = " WHERE (Programs.Program_ID = " . $access_array[0];
     foreach ($access_array as $program){
         $program_string .= " OR Programs.Program_ID = " . $program;
     }
-            $get_all_programs = "SELECT Session_ID, Session_Name, Name, Session_Names.End_Date FROM Session_Names INNER JOIN Programs ON Session_Names.Program_ID=Programs.Program_ID " . $program_string . " ORDER BY Name";
+    $program_string .= ")";
+    $old_date = date("Y-m-d", strtotime("-$num_days_hidden days"));
+    $get_all_programs_recent = "SELECT Session_ID, Session_Name, Name, Session_Names.End_Date FROM Session_Names INNER JOIN Programs ON Session_Names.Program_ID=Programs.Program_ID " . $program_string . " AND Session_Names.End_Date > '$old_date' ORDER BY Name";
 
-        include "../include/dbconnopen.php";

```

Comment:
I think the dark pink formatting there means a Tab was used instead of spaces, for indentation?  If so, please use spaces (okay to fix after merge though).

Reply:
No, that's a diff.  It means I de-indented it.  Github calls out diffs when it can figure out that you just removed or changed some specific characters with this coloring.

Reply:
Ooooooooooooooh.  Thanks.  I don't fully understand all the GitHub color codes; it's hard for me to remember color-based information -- much more of a distinct-symbols kind of person.